### PR TITLE
feat: Adding github issue labels and notes from the Office Hour

### PR DESCRIPTION
**Pull request by antonbabenko**
@terraform-aws-modules/triage-supporters WDYT about `labels` in `common.hcl`?

More specifically:
1. I am thinking to drop the description for all. It is rather obvious by the name anyway.
2. Can we drop some of the labels we won't be using so much?
3. Anything missing?

I have applied them all here - https://github.com/terraform-aws-modules/terraform-aws-pricing/labels

Inspiration for the data and colors:
* https://github.com/terraform-aws-modules/terraform-aws-eks/labels
* https://github.com/hashicorp/terraform-provider-aws/labels

### CODE REVIEW

File: github/repositories/terraform-aws-modules/common.hcl [hunk 1]
```diff
@@ -34,4 +34,87 @@ inputs = {
...
+      color       = "d66d0a"
+      description = "Breaking change features"
+    }
+
+    "bug" = {
+      color       = "d73a4a"
+      description = "Something isn't working"
+    }
+
+    "duplicate" = {
```
Comment (bryantbiggs) [inline github/repositories/terraform-aws-modules/common.hcl]: not sure about this one. I feel like if its a duplicate, we just close it out and reference the open issue/PR

File: github/repositories/terraform-aws-modules/common.hcl [hunk 2]
```diff
@@ -34,4 +34,87 @@ inputs = {
...
+      color       = "a5f7a7"
+      description = "Work in progress"
+    }
+
+    "wontfix" = {
+      color       = "ffffff"
+      description = "This will not be worked on"
+    }
+
+    "needs triage" = {
```
Comment (bryantbiggs) [inline github/repositories/terraform-aws-modules/common.hcl]: I think one additional label could be very useful - `needs example update` or something along those lines. we see it a lot where users will submit something but not update the example accordingly

Comment (daroga0002) [reply github/repositories/terraform-aws-modules/common.hcl]: I would put this in generic way "needs input"

Comment (antonbabenko) [reply github/repositories/terraform-aws-modules/common.hcl]: `needs example update`, `needs docs update` - something user will have to make.

How about adding a new one - `needs examples/docs update`?

File: github/repositories/terraform-aws-modules/common.hcl (hunk 1 above)
Comment (antonbabenko) [reply github/repositories/terraform-aws-modules/common.hcl]: I guess the same with `invalid`. We will just close it.

File: github/repositories/terraform-aws-modules/common.hcl (hunk 2 above)
Comment (antonbabenko) [reply github/repositories/terraform-aws-modules/common.hcl]: It is not `input` as such but when a user opens a PR with just changes in the terraform code - it is sometimes not enough. It is not `input`, but rather `output`... "needs output"? :)

Comment (bryantbiggs) [reply github/repositories/terraform-aws-modules/common.hcl]: ya, maybe combining with what @daroga0002 said - what about a generic `needs update`

Comment (daroga0002) [reply github/repositories/terraform-aws-modules/common.hcl]: I think "needs update” should be ok, as in issue or PR you can always specify directly what is required. Labels will be usefull in filtering issues, but too many of them can be also problematic.

I think currently this set should be optimal

Comment (antonbabenko) [reply github/repositories/terraform-aws-modules/common.hcl]: ok, `needs update`

**Comment by antonbabenko** [comment]
Applied. More feedback?

**Review by daroga0002 — APPROVED** [review]
For me looks good

**Comment by bryantbiggs** [comment]
looks great! :shipit:

**Pull request merged**